cppQML/README.md

```diff
@@ -17,11 +17,9 @@ You can install Qt features though **apt**, by installing packages like **qt6-ba
 
 You can open your application on Qt Creator and Qt Design Studio using the tasks **open-in-qt-creator** and **open-in-qt-design-studio**.
 
-Also in the **task runner**, it is possible to Start a GDB Server and perform the debug through there.
-
 Also, it is possible to debug the application (including the QML part) on Qt Creator, through the task **start-gdb-server-\<remote or local>-\${architecture}**. To do it, follow this steps:
 
- - Open the project on Qt Creator, through the **open-in-qt-creator** (it is necessary to open it through the task)
+ - Open the project on Qt Creator, through the **open-in-qt-creator-debug\${architecture}** task (**open-in-qt-creator** for local debug). It is necessary to open Qt Creator through this task.
  - Run the **start-gdb-server-\<remote or local>-\${architecture}** task
```

Comment:
I think it's worth mentioning here that this can take some time, and that these are the same tasks that are performed when using F5 from vs code, instead of attaching it to GDB it will only run the gdb server in the end.

I think it's also important to mention that you need to wait for the message (here a screenshot from VS Code would be very helpful):

```
Listening port 2320
```

before try to attach using the QtCreator

Reply:
Sure, I will update it here